docs/pages/accounts/audit-logs.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Audit logs
+description: A list of changes that happened to selected entities owned by an account.
```

Comment:
```suggestion
description: Learn how to track and analyze your account's activities by using a set of entities.
```

Not sure if this suggestion is the accurate description but i think the goal is to surface that an account's owner or admin can view their team members' activities. Let's update this description to convey that.